docs/compose-file.md

```diff
@@ -552,7 +552,7 @@ limit as an integer or soft/hard limits as a mapping.
         soft: 20000
         hard: 40000
 
-### volumes, volume\_driver
+### volumes
 
 Mount paths or named volumes, optionally specifying a path on the host machine
 (`HOST:CONTAINER`), or an access mode (`HOST:CONTAINER:ro`). Named volumes can
```

Comment:
There's a line just below this that says

> top-level `volumes` key, but this is optional - the Docker Engine will create the volume if it doesn't exist.

I think we need to move that to a **Note**: In V1 ...